library/src/scala/NamedTuple.scala

```diff
@@ -11,7 +11,7 @@ object NamedTuple:
   opaque type NamedTuple[N <: Tuple, +V <: Tuple] >: V <: AnyNamedTuple = V
 
   /** A type which is a supertype of all named tuples */
-  opaque type AnyNamedTuple = Any
+  opaque type AnyNamedTuple >: Tuple <: Product = Product
```

Comment:
We cannot change the right hand side anymore. It breaks binary and TASTy compatibility.

Reply:
If we assume 3.6.0 and 3.6.1 do no exist, then we're still in the RC period where modifications are allowed. We're currently at 3.6.2-RC2, and it's probably the last moment before signatures would be written in stone forever.

Reply:
Which makes me realize that we need another strategy to roll out new features. Hardly anybody uses them when they are experimental. And when we stabilize small nits like this one inevitably pop up. Which means we have only three choices:

 - we accept that it's imperfect and freeze anyway
 - we roll back, which means that we cannot get anything substantial into the language anymore, ever
 - we accept some breakage to fix nits such as this one, for a limited period.

I am not sure what to do here. Java has previews, would that help?

Reply:
It should be mentioned here that the SIP committee can approve a feature to be accepted, and it's the compiler's team decision when to actually merge it as non-experimental.